scripts/update_robots.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+#------------------------------------------------------------------------------
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2021 solidity contributors.
+#------------------------------------------------------------------------------
+
+set -eu
+
+repo_root="$(dirname "$0")/.."
+docs_static_path="docs/_static"
+robots_template_path="${repo_root}/${docs_static_path}/robots.txt.template"
```

Comment:
Well, the template is not a static file (we don't want it to be served along with the docs) so I'd put it in `docs/`.

I'd be tempted to put it in `_templates` but looks like that's only for templates that the Sphinx theme expects to already exist (see https://www.sphinx-doc.org/en/master/templating.html). Would be nice to reuse that mechanism but I think it won't work the way we want unfortunately.